great_expectations/rule_based_profiler/rule_based_profiler.py

```diff
@@ -153,12 +153,17 @@ def __init__(
 
         self._usage_statistics_handler = usage_statistics_handler
 
-        # Necessary to annotate ExpectationSuite during `run()`
+        # Necessary to annotation ExpectationSuite during `run()`
+        rule_to_load_into_citation: dict = {}
```

Comment:
@Shinnnyshinshin The initialization should not be needed, because `rule_to_load_into_citation` is set for every condition.  Thanks.

Reply:
This was negated by the fact that `add_rule()` now does the reconciliation step and loads the proper dict-formatted rule into the `profiler_config`. There is no need to check for the type :)